wdio.conf.js

```diff
@@ -317,6 +328,19 @@ if (isSauceEnabled) {
       commandTimeout: 600,
       maxDuration: 3600,
       seleniumVersion: '3.4.0',
+      extendedDebugging: true,
+      // extended debugging
+      'moz:firefoxOptions': {
+        args: [
+          '-start-debugger-server',
+          '9222'
+        ],
+        prefs: {
```

Comment:
Will this overwrite the options set in line 52?

Reply:
Yes I double checked before pushing because I wondered the same thing
`Object.assign(Obj1, Obj2, ..., source)`  so in our case `{}` gets overwritten by <-- `config` which gets overwritten by <-- whatever we put in there

Reply:
In that case, do we need all those options redefined here?

Reply:
For Firefox yes
There's no need for those options locally (as they only pertain to extendedDebugging/SauceLabs)
The options defined earlier are only necessary for local tests to run and are unnecessary for SauceLabs

Reply:
Awesome, sounds good!